Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -284,7 +286,7 @@ def to_class(self, response, ignored_keys: set | None = None, illegal_chars: set
 
 def http_request(uri: str, method: str, headers: dict = {},
                  body: dict = {}, params: dict = {}, files: dict | None = None, is_pcap: bool = False,
-                 is_xml: bool = False) -> Any:
+                 is_xml: bool = False, is_file: bool = False) -> Any:
```

Comment:
Let's just rename `is_pcap` here to `is_file` instead of having both, since they have the same purpose and `is_file` is more generalized.

(Make sure to change it in all the places this parameter is called, though)